tests/FunctionalTests/VerifyEmailHelperFunctionalTest.php

```diff
@@ -20,9 +20,14 @@
 use SymfonyCasts\Bundle\VerifyEmail\VerifyEmailHelperInterface;
 
 /**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ *
  * @group time-sensitive
+ *
+ * @internal
```

Comment:
I don't think we usually add `@internal` to tests. Tests I think are thought of as internal already - we wouldn't expect anyone to use tests directly from their code.